Source/WebCore/rendering/BorderPainter.cpp

```diff
@@ -1422,15 +1422,20 @@ Color BorderPainter::calculateBorderStyleColor(const BorderStyle& style, const B
 
     Operation operation = (side == BoxSide::Top || side == BoxSide::Left) == (style == BorderStyle::Inset) ? Darken : Lighten;
 
+    bool isVeryDarkColor = color.luminance() <= baseDarkColorLuminance;
```

Comment:
I’d prefer to find a way to share the code rather than repeating the same algorithm in two places.

Reply:
The code in `display` is currently dead code for future LFC, there is currently discussion to remove it.